app/helpers/ballots_helper.rb

```diff
@@ -4,4 +4,12 @@ def progress_bar_width(amount_available, amount_spent)
     (amount_spent / amount_available.to_f * 100).to_s + "%"
   end
 
+  def remaining_votes(ballot, group)
```

Comment:
Something is confusing here 🤔 We are looking at the number of votes per heading, but then looking at the votes in the group.

¿How about moving the `max_votes` to the `heading` class and comparing with the number of `votes` in the `heading`?

Reply:
Hmm...
What was I thinking...

Reply:
@voodoorai2000 Yah the code doesn't make sense. We can either fix this or as you suggested here, and hinted in a comment above move the max votes to the heading class. I am going to fix this for now. Let me know what you think is good.

Reply:
The remaining votes at the moment is:

          group.headings.count * group.number_votes_per_heading - ballot.investments.by_group(group.id).count